projects/js-themes-toolkit/packages/liferay-theme-tasks/theme/tasks/build/index.js

```diff
@@ -91,6 +94,7 @@ module.exports = function () {
 			'build:fix-at-directives',
 			'build:r2',
 			'build:copy:fontAwesome',
+			'build:minify:js',
```

Comment:
Wait until the very end and then minify all the things. Keep in mind that we don't have a babel step on our own, but people could configure it on their own, so chances are we're going to break things if that's the case (at least for sourcemaps).